flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorRecoveryITCase.java

```diff
@@ -58,7 +58,7 @@
 
 /** Recovery tests for {@link TaskExecutor}. */
 @ExtendWith(TestLoggerExtension.class)
-class TaskExecutorRecoveryTest {
+class TaskExecutorRecoveryITCase {
```

Comment:
I would like to double-check again whether this is really necessary.
Creating a TaskExecutor on it's own shouldn't be that expensive, and there are plenty of other unit tests doing that as well.

Reply:
Without this change, two out of two azure runs failed with OOM when running this test. Dunno why.

Reply:
`Could not allocate enough memory segments for NetworkBufferPool (required (Mb): 1024, allocated (Mb): 1022, missing (Mb): 2)`

Looks like we're using NETWORK_MEMORY_MAX (1g) by default, which pushes us a _bit_ over the memory budget when starting 2 TMs...

Reply:
https://issues.apache.org/jira/browse/FLINK-27351